### PR TITLE
Added Flask and directories

### PR DESCRIPTION
**Pull request by dcdarrell9**
# Context

Added a basic flask app that you can hit on port 8080 which will give a 'hello world'

# How to review

Get the branch, and play about with the system, I've used `pipenv` but any other suggestions are welcome.
I set myself a projects folder and then create an env var to point to that folder, so you'll see `${PROJECTS_HOME}/dir` in places, either set the var yourself, or just change it to your path.
There's no real substance to this, it's just a case of getting something on the board, and then looking to get things going. So make suggestions anywhere and we can raise issues / tasks.

# Checklist
- [x] Code has been tested
- [ ] Bug/Feature is tested/reviewed against acceptance criteria
- [x] Bug/Feature passes all tests and passes ci/cd analysis tools
- [ ] Code is review ready
- [ ] Bug/Feature is documented (Handover/Knowledge share included)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      - "8080:8080"
+#    labels:
+#      com.centurylinklabs.watchtower.enable: "true"
+    volumes:
+      - ${PROJECTS_HOME}/scratch-map
+    networks:
+      - scratch-net
+
+
+#  watchtower:
```
Comment (treilly94) [inline docker-compose.yml]: Maybe we should remove the watchtower stuff and chuck it in another branch

File: tests/app/test_info.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import unittest
```
Comment (treilly94) [inline tests/app/test_info.py]: In another file we're using pytest. Do we want to use both?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+MAINTAINER darrell.cox
+
+COPY . /app
+WORKDIR /app
+
+RUN pip install pipenv
+
+COPY Pipfile Pipfile
+COPY Pipfile.lock Pipfile.lock
```
[the remark this answers is not in the document]
Comment (dcdarrell9) [reply Dockerfile]: Yeah, suppose that depends on how much stuff we add? I prefer specific, but I guess we'll see in the future

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
+version: '3'
+
+services:
+
+  scratch-map:
+    container_name: scratch_map
+    build: ${PROJECTS_HOME}/scratch-map
```
[the remark this answers is not in the document]
Comment (dcdarrell9) [reply docker-compose.yml]: This is just a 'me' thing, I like to keep everything in one place, makes it easier then if I have different repos that work from the projects home var... Happy to take put though

File: docker-compose.yml (hunk 1 above)
Comment (dcdarrell9) [reply docker-compose.yml]: Yeah, that's fine, I just remember it being handy... Will remove

File: tests/app/test_info.py (hunk 2 above)
Comment (dcdarrell9) [reply tests/app/test_info.py]: I can't remember if you need Unittest to get pytest to work or not... Will need to check

Comment (emilioward) [reply tests/app/test_info.py]: No I believe they are 2 separate modules both independent. We just need to decide which one to go with. But i guess we can decide later when we write more code

File: docker-compose.yml (hunk 1 above)
Comment (emilioward) [reply docker-compose.yml:18]: I can remove this for @dcdarrell9 since he mentioned his computer needs to reinstall everything?

**Pull request merged**